### PR TITLE
Send code and language extension along with solution

### PR DESCRIPTION
**Pull request by piotrpalka**

### CODE REVIEW

File: atom-u2i-hackathon/lib/u2i-hackathon.coffee [hunk 1]
```diff
@@ -183,7 +183,9 @@ module.exports =
           "Error while checking your solution:\n" + error.message)
 
   onOutputs: (outputs) ->
-    requestSuccessful = @challengesService.checkSolution outputs
```
Comment (piotrpalka) [inline atom-u2i-hackathon/lib/u2i-hackathon.coffee:186]: We're not passing subtractedPoints to backend anywhere so we're always sending default value which is 0. I believe it's a bug or sending it does not make sense at all

File: atom-u2i-hackathon/lib/workspace-organizer.coffee [hunk 2]
```diff
@@ -71,9 +71,15 @@ class WorkspaceOrganizer
...
 
+  getCurrentSolution: ->
+    atom.workspace.getActiveTextEditor().getText()
+
+  chosenLanguageExtension: ->
+    @_languageChoiceView.getChosenLanguageExtension()
+
   _findExistingFileName: (challenge, teamId) ->
     fileExtension = @_languageChoiceView.getChosenLanguageExtension()
-    namePattern = new RegExp "^\\d{8}-\\d{6}-#{challenge.id}-#{teamId}.#{fileExtension}\\..+$"
```
Comment (piotrpalka) [inline atom-u2i-hackathon/lib/workspace-organizer.coffee:76]: This didn't work for me, new version works perfectly.

**Comment by piotrpalka** [comment]
Some tests are failing / have infinite loop, I wasn't able to make them work, however it's not caused by my changes.

File: atom-u2i-hackathon/lib/u2i-hackathon.coffee (hunk 1 above)
Comment (MichalPawlicki) [reply atom-u2i-hackathon/lib/u2i-hackathon.coffee:186]: There was a plan to write a plugin that subtracts points e.g. for each keystroke but it looks like we haven't implemented it.

**Review by MichalPawlicki — APPROVED** [review]

**Pull request merged**